### PR TITLE
fix(CX-1673): remove share button on BNMO flow

### PR DESCRIPTION
**Pull request by MounirDhahri**
The type of this PR is: **Bugfix**

This PR resolves [CX-1673]

### Description
- Remove share button on BNMO flow 

Group 1 (9)

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-1673]: https://artsyproduct.atlassian.net/browse/CX-1673

### CODE REVIEW

**Review by gkartalis — APPROVED** [review]
Looking good! 👍

File: Podfile.lock [hunk 1]
```diff
@@ -782,14 +782,7 @@ DEPENDENCIES:
...
 SPEC REPOS:
-  https://github.com/artsy/Specs.git:
-    - Aerodramus
-    - "Artsy+UIColors"
-    - "Artsy+UIFonts"
-    - "Artsy+UILabels"
-    - Artsy-UIButtons
-    - Extraction
-  trunk:
+  https://cdn.cocoapods.org/:
```
Comment (olerichter00) [inline Podfile.lock:785]: (question): are these changes intended?

**Review by olerichter00 — APPROVED** [review]
Looks good, just wondering about the `Podfile.lock` changes.

File: Podfile.lock (hunk 1 above)
Comment (MounirDhahri) [reply Podfile.lock:785]: they came after running `bundle execute pod install`
